frontend/smart-kickers-game/src/components/Game/GameStatistics/GameStatistics.js

```diff
@@ -2,8 +2,21 @@ import React from 'react';
 import './GameStatistics.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { Button } from '../../Button/Button.js';
+import { getStatistics } from '../../../apis/getStatistics.js';
+import { useEffect, useState } from 'react';
+import { TeamID } from '../../../constants/score.js';
 
 function GameStatistics({ finalScores, onNewGameRequested }) {
+  const [data, setData] = useState({});
+  useEffect(() => {
+    getStatistics().then((result) => {
+      if (result?.error) alert(result.error);
+      setData(result.data);
+    });
+  });
+  function returnFastestShot(teamID) {
+    return data.FastestShot.Team === teamID ? data.FastestShot.Speed.toFixed(2) + ' km/h' : '';
+  }
```

Comment:
Don't really understand the sense of displaying an empty string there.

Reply:
Changed with your suggestions.